zold_wallet/assets/fonts/Monospace.ttf

```diff
@@ -0,0 +1 @@
+Free download fonts at http://sharefonts.net
```

Comment:
@amorenew this is not the font file as I guess

Reply:
@ammaratef45 Yes it was a mistake I fixed it and amended the commit. check the last commit in this pull request.